### PR TITLE
Multiple images miniCPM-V-2_6

### PR DESCRIPTION
**Pull request by Wovchena**
TODO:
- [ ] Remove `ov::Core` from constructors.
- [ ] Hide files and API.

### CODE REVIEW

**Comment by ilya-lavrenov** [comment]
Do you have PR to your own repo to see only different introduced in current PR?

**Comment by Wovchena** [comment]
> Do you have PR to your own repo to see only different introduced in current PR?

1. https://github.com/Wovchena/openvino.genai-public/pull/65
2. https://github.com/Wovchena/openvino.genai-public/pull/67

File: .github/workflows/causal_lm_cpp.yml [hunk 1]
```diff
@@ -681,6 +681,43 @@ jobs:
...
+          && python -m pip install --upgrade-strategy eager ./thirdparty/openvino_tokenizers/[transformers] -r ./samples/requirements.txt --pre --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly
+      - run: >
+          source ./ov/setupvars.sh
+          && python ./samples/cpp/visual_language_chat/export_MiniCPM-V-2_6.py ./miniCPM-V-2_6/
+      - run: wget https://github.com/openvinotoolkit/openvino_notebooks/assets/29454499/d5fbbd1a-d484-415c-88cb-9986625b7b11
+      - run: >
+          source ./ov/setupvars.sh
+          && ./build/samples/cpp/visual_language_chat/visual_language_chat ./miniCPM-V-2_6/ d5fbbd1a-d484-415c-88cb-9986625b7b11
+          <<< $'What is on the image?\nWhat is special on the image?'
+        timeout-minutes: 110
```
Comment (ilya-lavrenov) [inline .github/workflows/causal_lm_cpp.yml:719]: maybe we can drop macOS since it takes almost 2h ?

**Review by ilya-lavrenov — APPROVED** [review]
Approved based on https://github.com/Wovchena/openvino.genai-public/pull/67

**Pull request merged**